DeAceroDLabs/marketplace-frontend
# DMP-643 save lat and lng and error shown

## What is done

[[DMP-643] Message error shown when map is not clicked and save lat and lng](https://deacero.atlassian.net/browse/DMP-643)

## How is done

- Added `errorMessage` to mapField so that it appears when map was not clicked.
![image](https://user-images.githubusercontent.com/81311646/198360186-82ec4f14-96c7-4877-816a-fd3f2ccee28d.png)

- Saved data:
![image](https://user-images.githubusercontent.com/81311646/198360026-e5f3480b-1ea5-4423-b2e8-db40507afbf2.png)


## How to test

1. Run the project via `npm start`
2. registrarse.
3. Fill first form.
4. Please do not click on map and click on `siguiente`. You should see error message on map.
